### PR TITLE
add queryError event

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: src/kuzzle.js [hunk 1]
```diff
@@ -57,7 +57,8 @@ module.exports = Kuzzle = function (host, options, cb) {
         jwtTokenExpired: {lastEmitted: null, listeners: []},
         loginAttempt: {lastEmitted: null, listeners: []},
         offlineQueuePush: {listeners: []},
-        offlineQueuePop: {listeners: []}
+        offlineQueuePop: {listeners: []},
+        queryError: {lastEmitted: null, listeners: []}
```
Comment (scottinet) [inline src/kuzzle.js]: Adding the `lastEmitted` property automatically prevents this event being fired more than once every `this.eventTimeout` ms (currently set to 200ms).

We may not want to throttle this.

Comment (stafyniaksacha) [inline src/kuzzle.js]: Done

**Comment by scottinet** [comment]
Do you plan to add this one to android and ios?

**Comment by j33f** [comment]
I need to have the actual pending PRs on iOS SDK to be merged then I'll figure out how to implement it

**Comment by stafyniaksacha** [comment]
It can be delayed, JIRA Ticket was written

**Pull request merged**